packages/contracts/CHANGELOG.md

```diff
@@ -5,21 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.3.1-rc0
+## v1.3.0
```

Comment:
Just to explicitly state what we are doing here now: 

We add the permission condition fix to the v1.3.0 release (although it was not present in any of the release candidates) because it doesn't affect the deployments that already happened everywhere.

If this is correct and we agree, all good.

Reply:
I don't know to be honest. I deployed the v1.3.0-rc0.4 tag. There this was from my point of view already present
https://github.com/aragon/osx/blob/6bc68ec65c6ebb0c4649bb2072a79a1acd054066/packages/contracts/src/core/permission/PermissionCondition.sol#L4

Additionally, the changelog is a pretty mess, checkout the pull #510 and see in the merge conflicts.

Reply:
Ok, then it is good that we clarify this explicitly here. I think what this PR does is correct. 
Explainer: Initially, we thought that we would release the missing virtual keyword `PermissionCondition` fix in a new patch release, but then things got delayed again and again. Now, it makes sense to put it into 1.3.0. This addition of `virtual` is formally not audited, but I think there isn't much to audit here.
Moreover, the `PermissionCondition` fix doesn't affect any plugin deployment we did in the past because we don't use a condition anywhere yet except a mocked one in our osx tests.